watertap/flowsheets/full_water_resource_recovery_facility/BSM2_P_extension.py

```diff
@@ -540,15 +546,15 @@ def scale_variables(m):
 
     # Apply scaling
     scale_variables(m)
-    iscale.calculate_scaling_factors(m.fs)
+    iscale.calculate_scaling_factors(m)
```

Comment:
How do we know whether to use m or m.fs.?

Reply:
In this case, there is no difference, since all units are built from the flowsheet `m.fs` level

Reply:
Ok, thanks for the clarification